tests/authentication_tests/test_shotgun_authenticator.py

```diff
@@ -12,12 +12,13 @@
 from mock import patch
 import os
 
-from tank_test.tank_test_base import *
+from tank_test.tank_test_base import setUpModule # noqa
```

Comment:
Changes in this file are to satisfy the pytest runner. Classes starting with Test but with an init method emits warnings.